src/Compilers/CSharp/Test/Semantic/Semantics/OperatorTests.cs

```diff
@@ -40,7 +39,51 @@ static void Main()
         Console.Write(rn);
     }
 }";
-            CompileAndVerify(source: source, expectedOutput: "TwoZero");
+            CompileAndVerify(source: source, expectedOutput: "TwoZero")
+                .VerifyOperationTree(@"
+Test.Main()
+  IBlockStatement (8 statements, 6 locals) (OperationKind.BlockStatement)
```

Comment:
Please provide comments on the dump, it probably seems verbose but I can't think of a smaller readable dump of the operation tree.